tools/sourcemap-tools/package.json

```diff
@@ -0,0 +1,40 @@
+{
+    "name": "@backtrace/sourcemap-tools",
+    "version": "0.0.1",
```

Comment:
Do we support all node envs? Otherwise we need engines support definition here

Reply:
Added.